test/dhcp_isc/integration_test.rb

```diff
@@ -39,3 +40,22 @@ def test_features
     end
   end
 end
+
+class DhcpApiValidIPTest < Test::Unit::TestCase
+  include Rack::Test::Methods
+
+  def app
+    app = Proxy::DhcpApi.new
+    app.helpers.server = @server
+    app
+  end
+
+  def test_validate_ip
+    @server = mock()
+    @server.stubs(:unused_ip).returns(true)
+    @server.expects(:validate_supported_address).with("192.168.122.0").returns("192.168.122.0")
+    @server.expects(:validate_supported_address).with("192.168.122.10", "192.168.122.20").returns("192.168.122.20")
+    get "/192.168.122.0/unused_ip?mac=01:02:03:04:05:06&from=192.168.122.10&to=192.168.122.20"
```

Comment:
While the repository is littered with bad examples, please use the example ranges according to https://community.theforeman.org/t/using-ips-and-domains-in-tests-documentation/16600.